src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2642 @@
+      this.supportUrl = supportUrl;
+      this.url = url;
+    }
+
+    public static Builder builder() {
+      return new com.stripe.param.AccountCreateParams.BusinessProfile.Builder();
+    }
+
+    public static class Builder {
+      private Long mcc;
+
+      private String name;
+
+      private String productDescription;
+
+      private String supportEmail;
+
+      private String supportPhone;
+
+      private String supportUrl;
+
+      private String url;
+
+      /** Finalize and obtain parameter instance from this builder. */
+      public BusinessProfile build() {
+        return new BusinessProfile(
+            this.mcc,
+            this.name,
+            this.productDescription,
+            this.supportEmail,
+            this.supportPhone,
+            this.supportUrl,
+            this.url);
+      }
+
+      /**
+       * The merchant category code for the account. MCCs are used to classify businesses based on
+       * the goods or services they provide.
+       */
+      public Builder setMcc(Long mcc) {
+        this.mcc = mcc;
+        return this;
+      }
+
+      /** The customer-facing business name. */
+      public Builder setName(String name) {
+        this.name = name;
+        return this;
+      }
+
+      /**
+       * Internal-only description of the product sold by, or service provided by, the business.
+       * Used by Stripe for risk and underwriting purposes.
+       */
+      public Builder setProductDescription(String productDescription) {
+        this.productDescription = productDescription;
+        return this;
+      }
+
+      /** A publicly available email address for sending support issues to. */
+      public Builder setSupportEmail(String supportEmail) {
+        this.supportEmail = supportEmail;
+        return this;
+      }
+
+      /** A publicly available phone number to call with support issues. */
+      public Builder setSupportPhone(String supportPhone) {
+        this.supportPhone = supportPhone;
+        return this;
+      }
+
+      /** A publicly available website for handling support issues. */
+      public Builder setSupportUrl(String supportUrl) {
+        this.supportUrl = supportUrl;
+        return this;
+      }
+
+      /** The business's publicly available website. */
+      public Builder setUrl(String url) {
+        this.url = url;
+        return this;
+      }
+    }
+  }
+
+  @Getter
+  public static class Company {
+    /** The company's primary address. */
+    @SerializedName("address")
+    Address address;
+
+    /** The Kana variation of the company's primary address (Japan only). */
+    @SerializedName("address_kana")
+    AddressKana addressKana;
+
+    /** The Kanji variation of the company's primary address (Japan only). */
+    @SerializedName("address_kanji")
+    AddressKanji addressKanji;
+
+    /** Whether information has been collected from the company's directors. */
+    @SerializedName("directors_provided")
+    Boolean directorsProvided;
```

Comment:
Random but wanted to make sure we won't have the same issue as we had in go for years where this sends `false` if it's not initialized. We had to move to pointers and that was a lot of work.

Reply:
Added test to verify this f8ede6b